cfpb/consumerfinance.gov
# Update `@cfpb/ccdb5-ui@2.11.4`

## Changes

- Update `@cfpb/ccdb5-ui@2.11.4`
- Adjust CCDB padding.


## How to test this PR

1. `yarn install && yarn build` and visit http://localhost:8000/data-research/consumer-complaints/search/ and compare the top and left/right padding to production. See screenshots for before/after.


## Screenshots

| Before    | After |
| -------- | ------- |
| <img width="1167" alt="Screenshot 2024-04-11 at 2 34 18 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/e20bb5c7-fa35-4118-a77e-35db1c74411a">  | <img width="1044" alt="Screenshot 2024-04-11 at 2 33 20 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/becc09cd-030d-48d4-8d42-10c7b39e7265">    |
| <img width="744" alt="Screenshot 2024-04-11 at 2 34 07 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/70296200-7f48-4aad-966b-5cf4bdcef836"> | <img width="811" alt="Screenshot 2024-04-11 at 2 33 31 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/02aeb080-65e9-4700-9c3d-7a0d7bf73b7e">    |
| <img width="505" alt="Screenshot 2024-04-11 at 2 33 58 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/8ee0c5f6-fa60-4cef-9a0d-800f885ef2c9">    | <img width="503" alt="Screenshot 2024-04-11 at 2 33 40 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/92739395-15a9-4a96-8805-a4213b436c68">    |
